### DISCUSSION TITLE
database_mysql

### DISCUSSION THREAD

**Discussion by FofoMalo** [symfony/symfony#45854] [Q&A] [answered]
Hi everybody,
hope your are fine and hope to find some help here!
it's my first project on symfony and i have an issue with my project : 
<img width="713" alt="symfony_issue" src="https://user-images.githubusercontent.com/71332281/160238230-e0bbb4ab-907d-48b6-a6f7-d09ba2a5c971.png">

- i change the url database for mysql.
- I create some entity.
my problem : are when i tape this command : php bin/console make:migration i got the issue.

Many thanks in advance to all yours answers.

thanks so much

**Comment by DigitalTimK** [comment]
Hey,
Looks like your need a password for your database. Did you defined the password in the `DATABASE_URL` in your `.env.local`-file?

`DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"`

**REMARK:** Some special characters in your password might need some special encoding

**Reply by FofoMalo** [reply]
thanks you @DigitalTimK  :)  i check it and i found the solution it's took me 2 days.
i hope ours discussion will help another person in the case

**Comment by FofoMalo** [comment, marked as answer]
i just resolve the situation. 
 clean your docker-compose file if you use mysql, cause it's start with Postgres
Don't hesitate to clean the file, and put the right port for you database.
In my case: 
version: '3.7'
services:
    database:
        image: 'mysql:latest'
        environment:
            MYSQL_ROOT_PASSWORD: password
            MYSQL_DATABASE: main
        ports:
            # To allow the host machine to access the ports below, modify the lines below.
            # For example, to allow the host to connect to port 3306 on the container, you would change
            # "3306" to "3306:3306". Where the first port is exposed to the host and the second is the container port.
            # See https://docs.docker.com/compose/compose-file/#ports for more information.
            - '3306'
        ###> symfony/mailer ###
    mailer:
        image: schickling/mailcatcher
        ports: [1025, 1080]
###< symfony/mailer ###
that'is my yaml file for docker-compose
